vizro-core/src/vizro/static/css/layout.css

```diff
@@ -87,7 +87,7 @@
   width: 336px;
 }
 
-.keyline {
+hr {
```

Comment:
Does it make sense to add a class (eg "vizro-hr") to every `html.Hr()` we use and target them with the `hr.vizro-hr` css selector. It might help if users add custom components that contain `html.Hr()` that are not affected by Vizro css.

Reply:
In this case, the other way around makes more sense :) In general, we should provide the visual default for the user to maintain visual consistency across the dashboards with our components. Hence, we define the default styling for all horizontal rulers via `hr`.

If a user wants to deviate from the default styling for a specific ruler, they will provide a class name to the ruler e.g. `html.Hr(className="my-ruler")` and provide their custom styling via `hr.my-ruler {}`.

Users who want to deviate from the default styling for all rulers can overwrite the `hr {}` in their custom assets folder. So here, we do not need to provide a className.

Reply:
Both approaches are correct, we just need to align and be consistent in the approach we want to push.

Reply:
(unresolving just so the comment shows, not because there's anything more to resolve here)

Reply:
We should be good! We already followed that approach for all typography elements and links, the horizontal ruler was the only element where we didn't follow it which was now fixed in this PR https://github.com/mckinsey/vizro/pull/91 :)

Reply:
Glad to see we're following this approach everywhere 😄.